OpenSim/Analyses/Test/testOutputReporter.cpp

```diff
@@ -183,15 +183,17 @@ void simulateMuscle(
     // 2. OUTPUTREPORTER SETUP: create and add the OutputReporter
     //==========================================================================
     OutputReporter* outputReporter = new OutputReporter(&model);
-    outputReporter->append_output_paths("kinetic_energy");
-    outputReporter->append_output_paths("jointset/slider/tx/value");
-    outputReporter->append_output_paths("bodyset/ball/linear_velocity");
-    outputReporter->append_output_paths("bodyset/ball/angular_acceleration");
-    outputReporter->append_output_paths("bodyset/ball/acceleration");
-    outputReporter->append_output_paths("jointset/slider/reaction_on_child");
+    outputReporter->append_output_paths("/|kinetic_energy");
```

Comment:
`/|k...` -> `k...`, or is this deliberate?

Reply:
Deliberate.

Reply:
`/` is the path for the `Model`, and `kinetic_energy` is one of its outputs.

Reply:
The advantage is that absolute paths are shorter and you can rename your model without needing to rename every absolute connectee path. We also had an inconsistency with state variable paths (didn't include the model name) and absolute component paths.

Reply:
I think it would be worth adding a comment in the doxygen. Users may edit in XML because they aren't comfortable programming. Something like `/|blah` isn't obvious and might send them to the documentation—and (hopefully) [this page](https://simtk.org/api_docs/opensim/api_docs/classOpenSim_1_1OutputReporter.html#a507dbb97cb33e0e858cfa521e655cf4e) where they would find only this: `"The names of Outputs to be reported. To select specific Component Outputs" ", provide its path name. For example, 'slider/tx/value' is an Output for " "the value of a Coordinate 'tx' belonging to the Joint 'slider'."`